exercises/practice/resistor-color-trio/.meta/example.gd

```diff
@@ -0,0 +1,18 @@
+func color_code(colors):
+	var colors_code = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
+
+	var units = {
+        1e9: "gigaohms",
```

Comment:
Much to my surprise, this file is correct by Godot Engine's standards, despite mixing spaces and tabs ... You learn something every new every day I guess ;)

Nevertheless, could you please replace spaces with tabs here as well? Just to keep things consistent ;) The rest of the PR looks good, once this is changed I will merge it :+1: